.storybook/main.js

```diff
@@ -14,5 +16,30 @@ module.exports = {
         controls: false,
       }
     }
-  ]
+  ],
+  webpackFinal: async (config) => {
+    // Make whatever fine-grained changes you need
+    config.module.rules.push({
+      test: /\.scss$/,
+      use: [{
+        loader: "style-loader"
+      },
+      {
+        loader: "css-loader",
+        options: {
+          importLoaders: 1,
+          modules: {
+            compileType: "icss"
+          }
+        }
+      },
+      {
+        loader: "sass-loader"
```

Comment:
Not entirely sure I understand this 🙈 

Does this enforce a specific processing order? Otherwise, why can't the `Sass` loader be part of the `css-loader` (see [css-loader docs](https://webpack.js.org/loaders/css-loader/#importloaders))?

Reply:
I’m no webpack expert, but this is what I know as our standard setup 🤔 Yes it’s ordered — sass gets compiled to css, then handed to the style-loader to insert into the page. I never knew css-loader could itself use loaders, I’ll look into it! What’s the advantage?

Reply:
Looking into this, there is a lot of confusion over what that option does! It’s not for compiling sass though — that should be done with sass-loader directly, as we’ve got here already. See https://github.com/webpack-contrib/css-loader/issues/228#issuecomment-312885975 and https://github.com/webpack-contrib/css-loader/issues/765 for some explanation, but the TL;DR is that it’s to help CSS-loader do the right thing when it encounters unresolved `@import` statements